src/component/header/Header.jsx

```diff
@@ -0,0 +1,86 @@
+import { useEffect, useState } from 'react';
+import headerJSON from '../../data/header.json';
+
+const Header = () => {
+    const [isChanged, setIsChanged] = useState(false);
+    const [isMenuOpen, setMenuOpen] = useState(false);
+
+    const toggleMenu = () => {
+        setMenuOpen(!isMenuOpen);
+        setIsChanged(!isChanged);
+    };
+
+    useEffect(() => {
+        const handleResize = () => {
+            if (window.innerWidth >= 768) {
+                setMenuOpen(false);
+                setIsChanged(false);
+            }
+        };
+
+        window.addEventListener('resize', handleResize);
+
+        return () => {
+            window.removeEventListener('resize', handleResize);
+        };
+    }, []);
+
+    return (
+        <div className="w-full fixed">
+            <div className="flex flex-row justify-between p-6 md:p-0  md:py-5 z-50 md:px-16 lg:px-20 xl:px-28 2xl:px-36 items-center  bg-white bg-opacity-50 backdrop-blur-md">
+                <div className="flex flex-row justify-center gap-x-3.5">
+                    <img src="/icons/y_logo.svg" alt="" className="w-4 h-6 2xl:w-6 2xl:h-8" />
+                    <img
+                        src="/icons/yume_labs_logo.svg"
+                        alt=""
+                        className="w-[144px] md:w-[120px] md:h-[20px] xl:w-[180px] 2xl:h-[25px] mt-1"
```

Comment:
Do not hardcode px use the nearest available. if its not possible then only hardcode pixel.